src/main/java/org/kainos/ea/db/RoleDao.java

```diff
@@ -0,0 +1,59 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.Role;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RoleDao {
+    private final DatabaseConnector databaseConnector = new DatabaseConnector();
```

Comment:
delete this line, database connection is passed in as variable to each method call.